sklearn/preprocessing/data.py

```diff
@@ -260,6 +260,13 @@ class MinMaxScaler(BaseEstimator, TransformerMixin):
         .. versionadded:: 0.17
            *data_range_*
 
+    n_samples_seen_ : int or array, shape (n_features,)
+        The number of samples processed by the estimator for each feature.
+        If there are not missing samples, the ``n_samples_seen`` will be an
```

Comment:
I do not think `MinMaxScaler` does anything special with missing samples and is always an integer.

Reply:
If the user doesn't define it, the parameter will be always an integer. It is not used in the fit or transformation. I thought it should be consistent with the StandardScaler which has those variations, but I can change it to the one I describe above.